.github/workflows/fossa.yml

```diff
@@ -0,0 +1,17 @@
+name: Fossa CLI
+on:
+  push:
+    branches:
+      - 'develop'
+      - 'release/**'
```

Comment:
Should these run on the pull request workflow, so that license issue could be caught early.
Not sure how this works, assuming this scans the code and dependencies for supported licenses, which are input somehow?

Reply:
The fossa cli github action if for pushing the analytics to the fossa server and not catching the issues realtime. So it cannot be a part of PR workflow.

Reply:
There's a different github app which can do this per PR, IIRC it's an app not an action, so probably doesn't need any yml action changes, just needs to be configured as such